### DISCUSSION TITLE
Regarding evaluation metrics for the calibration and validation periods, as well as issues related to model prediction.

### DISCUSSION THREAD

**Discussion by leemiaotang** [iiasa/CWatM#155] [Q&A]
Calibration and Validation Periods

In the video tutorial, the KGE and NS metrics for the period from 2000 to 2019 after calibration were presented, which are considered evaluation metrics for the calibration period. I would like to obtain the evaluation metrics for the validation period. Should I proceed as follows? For example, divide the observed data from 2000 to 2019 into two parts. The first part, from 2000 to 2010, is used as the observed data for the calibration period to calibrate the model. The second part, from 2011 to 2019, is used as the observed data for the validation period. Then, run the calibrated model to obtain simulated data for 2011-2019, and compare it with the observed data for 2011-2019 to calculate the validation period's KGE, NS, and other evaluation metrics manually.
![image-20240917104517890](https://github.com/user-attachments/assets/70e5c494-5534-4f2c-a67f-b56ab2453554)

Prediction Question: Future Trajectories

Is future prediction also done using the calibrated model? Is this process simply a matter of replacing past input data with CMIP6 data? Where can I obtain the code for the future trajectories that the professor used?

Thank you all for answering my questions!

**Comment by mikhailsmilovic** [comment]
Guten Morgen @miaotangya 

Your procedure for validation and calibration would work perfectly. The calibrated model can then be used for future scenarios. Yes, the process is simply a matter of replacing historical input data with CMIP6 data. Looking forward to your analyses!

We are using climatic data from ISIMIP, a model comparison Initiative.

The historical observed climatic data is from ISIMIP 3a.

The simulated climate scenarios are from  ISIMIP 3b. They are available here: https://www.isimip.org/gettingstarted/input-data-bias-adjustment/details/110/

Select (as in the attached image)
- Protocol, 
- Simulated climate, 
- ISIMIP 3b

![Screenshot_20240922-100302~2.png](https://github.com/user-attachments/assets/a129c06e-adf1-414a-a93d-72742f2bfdef)

The five datasets without (ocean) in the title are what we use in the future trajectory analyses:

- MRI-ESM2-0
- IPSL-CM6A-LR
- MPI-ESM1-2-HR
- UKESM1-0-LL
- GFDL-ESM4
